module/evdi_modeset.c

```diff
@@ -251,10 +260,33 @@ static void evdi_plane_atomic_update(struct drm_plane *plane,
 		    evdi_painter_needs_full_modeset(painter)) {
 
 			evdi_painter_set_scanout_buffer(painter, efb);
+
+#if KERNEL_VERSION(5, 0, 0) <= LINUX_VERSION_CODE
+			state->visible = true;
+			state->src.x1 = 0;
+			state->src.y1 = 0;
+			state->src.x2 = fb->width << 16;
+			state->src.y2 = fb->height << 16;
+
+			drm_atomic_helper_damage_iter_init(&iter, old_state, state);
+			while (drm_atomic_helper_damage_iter_next(&iter, &rect)) {
+				clip_rect.x1 = rect.x1;
+				clip_rect.y1 = rect.y1;
+				clip_rect.x2 = rect.x2;
+				clip_rect.y2 = rect.y2;
+				evdi_painter_mark_dirty(evdi, &clip_rect);
+				EVDI_DEBUG("Atomic update, dirty rect [%d, %d, %d, %d]\n",
```

Comment:
I think that evdi_painter_mark_dirty is already logging in dirty rects, and it is doing that in VERBOSE mode. Ihom this line is to be deleted.
Also default log mode is DEBUG so this log is to verbose.